docs/introduction.rst

```diff
@@ -0,0 +1,111 @@
+Introduction
+============
+
+The XAITK-Saliency toolkit implements a class of XAI algorithms known
+as `saliency algorithms`. The basic application pipeline is shown in Figure 1:
```

Comment:
Maybe change to "machine learning application pipeline"?

Reply:
sounds good